jxl/src/lib.rs

```diff
@@ -24,8 +24,6 @@ pub mod var_dct;
 // TODO: Move these to a more appropriate location.
 const BLOCK_DIM: usize = 8;
```

Comment:
Apparently, we have this one in `FrameHeader` above. Should we reduce to just one copy?

Reply:
Got rid of the ones in FrameHeader.